daemon/logs.go

```diff
@@ -78,7 +85,7 @@ func (daemon *Daemon) ContainerLogs(name string, config *ContainerLogsConfig) er
 			}
 		}
 	} else if err != nil {
-		logrus.Errorf("Error reading logs (json): %s", err)
+		logrus.Errorf("Error reading logs: %s", err)
```

Comment:
@cpuguy83 well actually there are far too many %s used in this file for errors (see above and below). I'm not gonna touch that for now.